R/coord-munch.r

```diff
@@ -9,11 +9,17 @@
 #'   All other variables are duplicated as needed.
 #' @param range Panel range specification.
 #' @param segment_length Target segment length
+#' @param make_closed Whether data should be converted to closed polygons before
+#'   munching.
 #' @keywords internal
 #' @export
-coord_munch <- function(coord, data, range, segment_length = 0.01) {
+coord_munch <- function(coord, data, range, segment_length = 0.01, make_closed = FALSE) {
```

Comment:
`make_closed` is probably not the right name here... the coordinates are closed per definition so something like `is_closed()` is better

Reply:
So should I just flip the logic around and use `is_closed = TRUE` as default and then `geom_polygon()` uses `is_closed = FALSE`?

Reply:
No, `is_closed` describes whether the coordinates should be considered closed, which would be true for polygons... so it is simply a change in argument name and nothing more